### ISSUE TITLE
Backup.db

### ISSUE THREAD

**Issue by alucard73** [justMaku/gpghax#18]
Hello, I have a minor concern, the scrip must have a default because I have no backup.db as expected in my usb key. I only have the custom.db but can not create when I put my usb the backup.db. A person would have it in original to provide me?

**Comment by justMaku** [comment]
This "safety" mechanism was added later on so if you used the original payload it might be missing. Luckily the system has a backup database you can copy:

You need to check which of those databases corresponds to your region and modify the payload to copy it instead of the custom db.

```
/usr/sony/share/data/databases/regional_J.db
/usr/sony/share/data/databases/regional_UC.db
/usr/sony/share/data/databases/regional_EK.db
/usr/sony/share/data/databases/regional_ET.db
```

TLDR:
Modify the lolhack.sh and add:

```
cp /usr/sony/share/data/databases/regional_* /media
```

Run the hack, open the usb drive on the pc, check with SQL Editor which database is your local one, and rename it to `custom.db` and put in `games` directory and run the hack again. It should restore the default database.
